submissions/kt-std/memory-pair-game/js/script.js

```diff
@@ -0,0 +1,145 @@
+const container = document.querySelector(".container"),
+	numCards = 9,
+	cardsData = ((assetsPath) => {
+		const arr = [];
+		for (let i = 1; i <= numCards; i++) {
+			const card = {path: `${assetsPath}p${i}.svg`, name: `p${i}`};
+			arr.push(card, card);
+		}
+		return arr;
+	})("assets/");
+
+let checkedCards = [],
+	checkedCardsContainers = [],
+	shuffledCards = shuffleArray(cardsData),
+	foundMatches = [];
+
+function shuffleArray(o) {
+	for (
+		var j, x, i = o.length;
```

Comment:
why `var`?

Reply:
`var` is not needed, so I've updated this one with the `let`